detectable/src/main/java/com/synopsys/integration/detectable/detectables/bazel/BazelExtractor.java

```diff
@@ -75,30 +98,20 @@ private Extraction buildResults(List<Dependency> aggregatedDependencies, String
     }
 
     @NotNull
-    private List<Dependency> collectDependencies(Pipelines pipelines, Set<WorkspaceRule> workspaceRules) throws IntegrationException {
+    private List<Dependency> collectDependencies(Pipelines pipelines, Set<WorkspaceRule> workspaceRules) throws DetectableException, ExecutableFailedException {
```

Comment:
I question the utility of `DependencyTransformer` as is. I would move this method in there and rename it to something like `PipelineDependencyGraphTransformer`

Reply:
Certainly could. And I think there is value in getting that out of the extractors if we want them to stay simple. That moves basically everything (it adds command executing, parsing, and the first phase of transforming) into that class. @JakeMathews  At that point I think we'd want to call it something other than Transformer, wouldn't we?